### PR TITLE
feature: incident list page (Issue No: #46)

### PR DESCRIPTION
**Pull request by Shabab007**
Screenshot 2024-09-04 at 5 30 56 PM

### CODE REVIEW

File: web_app/hazard-reporting-map/pages/incidents/index.tsx [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+              </div>
+            </div>
+
+            {/* data table */}
+            <div className="rounded-md">
+              <DataTable columns={columns} loading={false} data={dataSlice} />
+            </div>
+            {/* pagination */}
+            <div className=" flex justify-center md:justify-end px-4 py-6 md:py-2 text-xs items-center mt-8  rounded ">
+              <div className="">
```
Comment (Muttakee31) [inline web_app/hazard-reporting-map/pages/incidents/index.tsx:340]: we can remove the classname for now if it is empty.

File: web_app/hazard-reporting-map/pages/incidents/index.tsx [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+  Plus,
+  Search,
+  Trash,
+  Trash2,
+  Eye
+} from 'lucide-react';
+import { useState } from 'react';
+import ReactPaginate from 'react-paginate';
+
+export default function IncidentsLisPage() {
```
Comment (Muttakee31) [inline web_app/hazard-reporting-map/pages/incidents/index.tsx:34]: We can rename the class to `IncidentsListPage`. I think there is a typo here.

Comment (Shabab007) [reply web_app/hazard-reporting-map/pages/incidents/index.tsx]: sure. done

File: web_app/hazard-reporting-map/pages/incidents/index.tsx (hunk 1 above)
Comment (Shabab007) [reply web_app/hazard-reporting-map/pages/incidents/index.tsx]: done

**Review by skfarhad — APPROVED** [review]
Looks good! Nice work!

**Pull request merged**